hbase-mapreduce/src/test/java/org/apache/hadoop/hbase/mapreduce/TestTableSnapshotInputFormat.java

```diff
@@ -304,6 +305,56 @@ public void testWithMockedMapReduceWithNoStartRowStopRow() throws Exception {
     }
   }
 
+  @Test
+  public void testScanLimit() throws Exception {
+    setupCluster();
```

Comment:
Does it make sense to move this one to a test initializer method? I see it used in several tests in here.

Reply:
It is better to submit another PR to fix?after this one merged, I will amend it

Reply:
It's up to you. We can also do it in a separate ticket.

Reply:
Would you like to help merge this PR?  Thanks